core/async/src/multithread/runtime_handle.rs

```diff
@@ -30,10 +37,14 @@ where
 }
 
 /// See ActorSystem::spawn_multithread_actor.
+///
+/// The `system_cancellation_signal_preventer` is an optional sender that can be used to disable
```

Comment:
nit: Change variable name to `cancellation_signal_holder`. Mention this is the same Sender for `cancellation_signal`